ThunkLibs/GuestLibs/CMakeLists.txt

```
@@ -28,6 +28,7 @@ function(generate NAME SOURCE_FILE)
   # Shorthand for the include directories added after calling this function.
```

Since we now can propagate definitions, we can remove the manual `-DGUEST_THUNK_LIBRARY` switch below and in `add_guest_lib` and instead use this:

```suggestion
  target_compile_definitions(${NAME}-guest-deps INTERFACE GUEST_THUNK_LIBRARY)
  # Shorthand for the include directories added after calling this function.
```

---

Good shout, resolved.